tests/calc/test_utils.py

```diff
@@ -107,3 +111,104 @@ def test_convert_units():
     df = pd.DataFrame({'Val': [-40, 0, 10]})
     np.testing.assert_allclose(utils.convert_units('degC', 'degF', df).Val[0], -40)
     np.testing.assert_allclose(utils.convert_units('degC', 'degF', df).Val[1], 32)
+
+
+def test_to_altitude():
```

Comment:
I think this test does the right stuff, but it is going to get a bit harder to maintain as time goes on. What do you think about updating it to use parameterization to take in an input dataframe, parameters, expected result, and maybe an error message, and then essentially looping through the code?
Something like the 2nd example in https://hutchinson-digital.atlassian.net/wiki/spaces/GE/pages/167575568/Best+Practices+for+Python+Testing#Testing-Exceptions we can discuss if it is unclear